hbase-server/src/main/java/org/apache/hadoop/hbase/namequeues/NamedQueuePayload.java

```diff
@@ -30,7 +30,8 @@
 public class NamedQueuePayload {
 
   public enum NamedQueueEvent {
-    SLOW_LOG
+    SLOW_LOG,
```

Comment:
I wish I had caught this earlier. 
If we have to use an Enum here, add a constructor that defines an ordinal for each type so we can maintain compatibility by instantiating by our ordinal, e.g.

NamedQueueEvent(int ordinal) { ... }

SLOW_LOG(1),
BALANCE_DECISION(2),.
...